src/utils/language/nb.ts

```diff
@@ -1,18 +1,30 @@
+/**
+ * The array/tuple notation is used to specify a second value which is the aria-label for form control elements.
+ */
+
 export const nb = {
   allCatalogs: 'Alle kataloger',
   catalogType: 'Datasettkatalog',
+  dataVillage: ['Datalandsbyen', 'Lenke til Datalandsbyen'],
+  hostData: ['Høste data', 'Lenke til å høste data'],
```

Comment:
Vet ikke om jeg er så fan av navnet her, resten er skrevet på Engelsk, så mer lesbart navn ville kanskje vært `harvestData` ? Men er kanskje brukt konsekvent på flere steder, ser feks at det også bruker for `hostDataset`. Hva tenker du?

Reply:
Bra tenkt.  Skal erstatte **host** med **harvest** i alle lignende tilfeller.